plugins/reporters/web-app-template/src/components/AboutModal.jsx

```diff
@@ -174,8 +174,10 @@ const AboutModal = ({ webAppOrtResult }) => {
                                 </p>
                                 <p>
                                     Licensed under Apache License, Version 2.0 (SPDX: Apache-2.0) but also includes
-                                    third-party software components under other open source licenses.
-                                    See OSS Review Toolkit code repository for further details.
+                                    third-party software components under other open source licenses.<br/>
+                                    See <a href="https://github.com/oss-review-toolkit/ort">
```

Comment:
"See *the*..."

Reply:
And maybe link to https://oss-review-toolkit.org/ort/docs/license instead?

Reply:
Though about this went I made the commit but because states "but also includes third-party software components under other open source licenses" I thought it's safer/more future proof to point to the code repo at root level instead of LICENSE or LICENSES dir.

Reply:
I was just thinking that a more "end-user friendly" link would be better to use in the web-app.

Reply:
I only added the link as quick fix so people could more easily find the ORT code repo (based on feedback I received). I am not happy with current About and in follow up PR About Model it will be overhauled to include ORT version, ORT component times. Can we not merge and revisit this in next webapp PR?

Reply:
> Can we not merge and revisit this in next webapp PR?

Sure, that alone should not be a blocker.